Sources/AirbnbSwiftFormatTool/AirbnbSwiftFormatTool.swift

```diff
@@ -101,8 +101,6 @@ struct AirbnbSwiftFormatTool: ParsableCommand {
       "--config", swiftLintConfig,
       // Required for SwiftLint to emit a non-zero exit code on lint failure
       "--strict",
-      // This flag is required when invoking SwiftLint from an SPM plugin, due to sandboxing
-      "--in-process-sourcekit",
```

Comment:
This flag is no longer necessary:
 > warning: The --in-process-sourcekit option is deprecated. SwiftLint now always uses an in-process SourceKit.